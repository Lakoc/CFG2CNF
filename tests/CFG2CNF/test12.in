S,A,B
a,b,c
S
S->abcAb
S->A
A->B
B->A