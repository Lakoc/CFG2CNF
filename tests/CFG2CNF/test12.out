S,A,B,a',b',c',<bcAb>,<cAb>,<Ab>
a,b,c
S
S->a'<bcAb>
<bcAb>->b'<cAb>
<cAb>->c'<Ab>
<Ab>->Ab'
b'->b
c'->c
a'->a